doc/Usage.md

```diff
@@ -0,0 +1,37 @@
+# Using Custom Student Verification App
+
+## Theme Customization
+
+To use this app you need to customize/override some templates in your custom theme. To know more about edX comprehensive theming, [check here](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/theming.html).
```

Comment:
@shimulch, this link points to the documentation on ecommerce theming and not the LMS theming.

Reply:
Fixed, sorry for the mistake.